python/fedml/api/modules/run.py

```diff
@@ -11,8 +11,8 @@
 
 
 class RunLogResult(object):
-    def __init__(self, run_status: str = None, total_log_lines: int = 0, total_log_pages: int = 0,
-                 log_line_list: List[str] = list(), run_logs: FedMLRunLogModelList = None):
+    def __init__(self, run_status: RunStatus = None, total_log_lines: int = 0, total_log_pages: int = 0,
+                 log_line_list: List[str] = None, run_logs: FedMLRunLogModelList = None):
```

Comment:
If `log_line_list: List[str] = None` is a nullable object, then the type hint should be updated to `log_line_list: Optional[List[str]] = None`

Reply:
Done. Thanks!